spec/cucumber/cli/profile_loader_spec.rb

```diff
@@ -42,6 +42,25 @@ def loader
         given_cucumber_yml_defined_as yml
         expect(loader.args_from('default')).to eq ['--format','pretty','features/sync_imap_mailbox.feature:16:22']
       end
+
+      it 'correctly parses a profile that uses tag expressions (with double quotes)' do
```

Comment:
I'm wondering if this test name is misleading. It seems to me that the bug we've found is to do with parsing *any* switches given after the feature file paths, and that tag expressions just happen to be the example we've found of that general defective behaviour. Am I right?